fendermint/app/src/options/key.rs

```diff
@@ -21,6 +23,19 @@ pub struct KeyArgs {
     pub command: KeyCommands,
 }
 
+#[derive(Args, Debug)]
+pub struct AddPeer {
+    /// The path to node key file.
```

Comment:
```suggestion
    /// The path to a CometBFT node key file.
```
Let's give an example of its default location, so people have an idea where to look to understand what this is about.